### PR TITLE
NFT-335: Optimism config

### PR DESCRIPTION
**Pull request by adamgobes**

### CODE REVIEW

File: lib/config.ts [hunk 1]
```diff
@@ -55,12 +55,34 @@ const ethereum = {
...
+  ...baseConfig,
+  centerNetwork: '',
+  nftBackedLoansSubgraph:
+    'https://api.thegraph.com/subgraphs/name/with-backed/backed-protocol-optimism',
+  nftSalesSubgraph:
+    'https://api.thegraph.com/subgraphs/name/adamgobes/nft-sales-indexer',
+  eip721Subgraph:
+    'https://api.thegraph.com/subgraphs/name/optimism-erc721-subgraph',
+  infuraId: '54c753f04ec64374aa679e383e7f84d5',
+  pirschCode: '82ZKa76zuhAYZcSHM5QpOGU8U4hY5f1l',
```
Comment (adamgobes) [inline lib/config.ts:68]: @cnasc do we want a different pirsch code? assume not

File: lib/config.ts [hunk 2]
```diff
@@ -55,12 +55,34 @@ const ethereum = {
...
 
+const optimism = {
+  ...baseConfig,
+  centerNetwork: '',
+  nftBackedLoansSubgraph:
+    'https://api.thegraph.com/subgraphs/name/with-backed/backed-protocol-optimism',
+  nftSalesSubgraph:
+    'https://api.thegraph.com/subgraphs/name/adamgobes/nft-sales-indexer',
+  eip721Subgraph:
+    'https://api.thegraph.com/subgraphs/name/optimism-erc721-subgraph',
```
Comment (cnasc) [inline lib/config.ts:66]: @adamgobes let's double check this, doesn't seem to be valid

Comment (adamgobes) [reply lib/config.ts:66]: ah oops had the change staged but not commited 😭

**Review by cnasc — APPROVED** [review]
:shipit:

**Pull request merged**